cms/templates/partials/navigation.html

```diff
@@ -2,103 +2,105 @@
 CODEPEN: https://codepen.io/vixxofsweden/pen/xxGGYOE
 -->
 <!-- needs refining and styling & Fix mobile-->
-<div class="nhsuk-width-container">
-  <div class="nav mega_nav_wrapper">
-      <nav id="header-navigation" class="mega_nav" role="navigation" aria-label="Primary navigation" aria-labbeledby="label-navigation">
-          <a href="javascript:void(0);" class="mobile-menu-trigger">Open mobile menu</a>
-          <ul class="menu menu-bar" aria-hidden="false" role="menubar">
-              <li>
-                  <a href="javascript:void(0);" class="menu-link menu-bar-link" aria-haspopup="true">
```

Comment:
Actually I'm not sure what is this doing here so I'm gonna double check if it's not breaking the navigation working